hokusai/cli/production.py

```diff
@@ -4,6 +4,7 @@
 
 from hokusai.cli.base import base
 from hokusai.lib.common import set_verbosity, CONTEXT_SETTINGS
+from hokusai.lib.default_branch import detect_branch
```

Comment:
Nitpick: it might make more sense to call this file something like `branch_detection.py`; `default_branch` is a bit confusing because the default branch is just one of the possible return values.